### PR TITLE
real: Add version 0.2.0

### PR DESCRIPTION
**Pull request by ghost-420**
Note: "bin" and "shortcuts" done nothing while I've tested.

### CODE REVIEW

File: bucket/real.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "version": "0.2.0",
+    "description": "Reduce audio latency on Windows 10",
```
Comment (Ash258) [inline bucket/real.json:3]:
```suggestion
    "description": "Reduce audio latency on Windows 10.",
```

File: bucket/real.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            [
+                "REAL.exe",
+                "Reduce Audio Latency",
+                "--tray"
+            ]
+        ],
+        "checkver": {
+            "github": "https://github.com/miniant-git/REAL"
+        }
+    }
```
Comment (Ash258) [inline bucket/real.json:24]:
```suggestion
    "suggest": {
        "vcredist": "vcredist2017"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/miniant-git/REAL/releases/download/updater-v2/REAL.exe",
            "hash": "d9767fd0e29a9fcdd274282de81c82d7b8fb16f268fa2a7cec50c01d0dd3c4ad"
        }
    },
    "pre_install": "if ([Environment]::OSVersion.Version.Major -lt 10) { throw 'Windows 10 is required' }",
    "bin": "REAL.exe",
    "shortcuts": [
        [
            "REAL.exe",
            "Reduce Audio Latency",
            "--tray"
        ]
    ],
    "checkver": {
        "github": "https://github.com/miniant-git/REAL",
        "regex": ">REAL\\s+v([\\d.]+)</"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/miniant-git/REAL/releases/download/updater-v2/REAL.exe"
            }
        }
    }
```

**Comment by ghost-420** [comment]
Works fully as intended after applying the suggested changes.

File: bucket/real.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+    "version": "0.2.0",
+    "description": "Reduce audio latency on Windows 10.",
+    "homepage": "https://github.com/miniant-git/REAL",
+    "license": "MIT",
+    "notes": [
+        "Windows start menu search for 'real' or 'REAL' launches without parameters; for reducing audio latency only when needed to lower power usage.",
+        "'Reduce Audio Latency' shortcut launches with parameters to hide in the system tray; this makes it more difficult to stop the audio latency reduction.",
+        "-----"
+    ],
```
Comment (Ash258) [inline bucket/real.json:10]:
```suggestion
```

Comment (ghost-420) [reply bucket/real.json:10]: Why?

File: bucket/real.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.2.0",
+    "description": "Reduce audio latency on Windows 10.",
+    "homepage": "https://github.com/miniant-git/REAL",
+    "license": "MIT",
+    "notes": [
+        "The 'Reduce Audio Latency' shortcut opens REAL with custom launch parameters, to run it in the background; uses more battery/power usage while active.",
+        "-----"
+    ],
```
Comment (Ash258) [inline bucket/real.json:9]:
```suggestion
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Pull request merged**